e2e/testcases/stress_test.go

```diff
@@ -261,6 +265,86 @@ func TestStressLargeRequest(t *testing.T) {
 	}
 }
 
+func TestStress100CRDs(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured, ntopts.StressTest,
+		ntopts.WithReconcileTimeout(configsync.DefaultReconcileTimeout))
+
+	// Apply 100 small CRDs on the cluster to make sure Config Sync can still work.
+	for i := 1; i <= 100; i++ {
+		crd := fakeCRD("anvil", fmt.Sprintf("acme-%d.com", i))
```

Comment:
FWIW, this creates 100 API groups too, not just 100 resources. This will cause API Discovery to make 100 different API calls on older k8s clusters (one per group), but I assume that's intentional. Using the same group might test the memory usage without the api calls, but I don't know if that's necessary. This way is more stress.